ports/libsimpleservo/api/src/lib.rs

```diff
@@ -516,6 +526,28 @@ impl ServoGlue {
                     }
                     self.events.push(WindowEvent::SelectBrowser(new_browser_id));
                 },
+                EmbedderMsg::GetClipboardContents(sender) => {
+                    let contents = match self.clipboard_ctx {
```

Comment:
Instead of directly manipulating the clipboard for the GetClipboardContents and SetClipboardContents messages, let's add new callbacks to HostCallbacks (`get_clipboard_contents()` can return an Option<String> and `set_clipboard_contents` can accept a String) and call those instead. See how EmbedderMsg::HistoryChanged is handled earlier.